### PR TITLE
Add unit test for user listing page

### PR DESCRIPTION
**Pull request by yuxitang-amzn**
*Issue #, if available:*

*Description of changes:*
Add unit test for user listing page.

Coverage report:
**Before:**
% yarn test:jest --coverage
yarn run v1.22.4
$ NODE_PATH=../../node_modules ../../node_modules/.bin/jest --config ./test/jest.config.js --coverage
 PASS  public/apps/configuration/panels/role-edit/tenant-panel.test.tsx (13.711s)
 PASS  public/apps/configuration/panels/role-edit/role-edit.test.tsx (13.799s)
 PASS  public/apps/configuration/panels/role-list.test.tsx (14.341s)
 PASS  public/apps/configuration/utils/toast-utils.test.tsx
 PASS  server/auth/auth_handler_factory.test.ts
 PASS  public/apps/configuration/panels/role-edit/index-permission-panel.test.tsx
 PASS  public/apps/configuration/utils/tenant-utils.test.tsx
 PASS  public/apps/configuration/utils/role-list-utils.test.tsx
 PASS  public/apps/configuration/panels/role-edit/cluster-permission-panel.test.tsx
 FAIL  test/jest_integration/security_entity_api.test.ts

File                                                 | % Stmts | % Branch | % Funcs | % Lines | Uncovered Line #s
-----------------------------------------------------|---------|----------|---------|---------|---------------------------------------------
All files                                            |   11.58 |    10.24 |   13.25 |    11.3 |
 build_tools                                         |       0 |        0 |       0 |       0 |
  build_plugin.js                                    |       0 |        0 |       0 |       0 | 16-153
 common                                              |     100 |      100 |     100 |     100 |
  index.ts                                           |     100 |      100 |     100 |     100 |
 public                                              |       0 |        0 |       0 |       0 |
  index.ts                                           |       0 |      100 |       0 |       0 | 24
  plugin.ts                                          |       0 |        0 |       0 |       0 | 38-122
  types.ts                                           |       0 |        0 |       0 |       0 |
 public/apps                                         |       0 |      100 |     100 |       0 |
  apps-constants.tsx                                 |       0 |      100 |     100 |       0 | 18-21
  types.ts                                           |       0 |        0 |       0 |       0 |
 public/apps/account                                 |       0 |        0 |       0 |       0 |
  account-app.tsx                                    |       0 |        0 |       0 |       0 | 24-40
  account-nav-button.tsx                             |       0 |        0 |       0 |       0 | 45-134
  constants.tsx                                      |       0 |      100 |     100 |       0 | 19
  log-out-button.tsx                                 |       0 |        0 |       0 |       0 | 22-37
  password-reset-panel.tsx                           |       0 |      100 |       0 |       0 | 41-138
  role-info-panel.tsx                                |       0 |        0 |       0 |       0 | 29-73
  tenant-selection-page.tsx                          |       0 |        0 |       0 |       0 | 24-51
  tenant-switch-panel.tsx                            |       0 |        0 |       0 |       0 | 49-249
  types.ts                                           |       0 |        0 |       0 |       0 |
  utils.tsx                                          |       0 |        0 |       0 |       0 | 22-39
 public/apps/configuration                           |   26.87 |        0 |       0 |   27.69 |
  app-router.tsx                                     |       0 |        0 |       0 |       0 | 42-235
  configuration-app.tsx                              |       0 |      100 |       0 |       0 | 30-32
  constants.tsx                                      |     100 |      100 |     100 |     100 |
  cross-page-toast.tsx                               |       0 |        0 |       0 |       0 | 24-34
  types.ts                                           |       0 |        0 |       0 |       0 |
  ui-constants.tsx                                   |     100 |      100 |     100 |     100 |
 public/apps/configuration/panels                    |   24.74 |    18.18 |    7.89 |   25.81 |
  expression-modal.tsx                               |       0 |        0 |       0 |       0 | 28-54
  get-started.tsx                                    |       0 |      100 |       0 |       0 | 37-200
  nav-panel.tsx                                      |       0 |      100 |       0 |       0 | 21-36
  role-list.tsx                                      |   64.86 |       50 |      30 |   66.67 | 55,119-130,143-167,177
  user-list.tsx                                      |       0 |        0 |       0 |       0 | 51-225
 public/apps/configuration/panels/audit-logging      |       0 |        0 |       0 |       0 |
  audit-logging-edit-settings.tsx                    |       0 |        0 |       0 |       0 | 45-240
  audit-logging.tsx                                  |       0 |        0 |       0 |       0 | 52-221
  code-editor.tsx                                    |       0 |        0 |       0 |       0 | 27-45
  constants.tsx                                      |       0 |      100 |     100 |       0 | 16-339
  edit-setting-group.tsx                             |       0 |        0 |       0 |       0 | 34-152
  types.tsx                                          |       0 |        0 |       0 |       0 |
  view-setting-group.tsx                             |       0 |        0 |       0 |       0 | 32-62
 public/apps/configuration/panels/auth-view          |       0 |        0 |       0 |       0 |
  auth-view.tsx                                      |       0 |        0 |       0 |       0 | 25-87
  authentication-sequence-panel.tsx                  |       0 |        0 |       0 |       0 | 24-99
  authorization-panel.tsx                            |       0 |        0 |       0 |       0 | 24-87
 public/apps/configuration/panels/internal-user-edit |       0 |        0 |       0 |       0 |
  attribute-panel.tsx                                |       0 |        0 |       0 |       0 | 37-125
  internal-user-edit.tsx                             |       0 |        0 |       0 |       0 | 50-180
  types.tsx                                          |       0 |        0 |       0 |       0 |
 public/apps/configuration/panels/permission-list    |       0 |        0 |       0 |       0 |
  edit-modal.tsx                                     |       0 |        0 |       0 |       0 | 43-97
  permission-list.tsx                                |       0 |        0 |       0 |       0 | 63-321
 public/apps/configuration/panels/permission-tree    |       0 |        0 |       0 |       0 |
  index.ts                                           |       0 |        0 |       0 |       0 |
  permission-tree.tsx                                |       0 |        0 |       0 |       0 | 23-52
 public/apps/configuration/panels/role-edit          |   87.61 |       80 |   89.58 |   87.27 |
  cluster-permission-panel.tsx                       |     100 |      100 |     100 |     100 |
  constant.tsx                                       |     100 |      100 |     100 |     100 |
  index-permission-panel.tsx                         |   93.75 |    83.33 |   90.91 |    93.1 | 271-289
  role-edit.tsx                                      |   79.31 |       75 |   76.92 |   79.31 | 107-111,126-127,140-141,152,155,172-173,249
  tenant-panel.tsx                                   |     100 |      100 |     100 |     100 |
  types.tsx                                          |       0 |        0 |       0 |       0 |
 public/apps/configuration/panels/role-mapping       |       0 |        0 |       0 |       0 |
  RoleEditMappedUser.tsx                             |       0 |        0 |       0 |       0 | 51-162
  external-identities-panel.tsx                      |       0 |        0 |       0 |       0 | 38-106
  internal-users-panel.tsx                           |       0 |      100 |       0 |       0 | 30-31
  types.tsx                                          |       0 |        0 |       0 |       0 |
 public/apps/configuration/panels/role-view          |       0 |        0 |       0 |       0 |
  cluster-permission-panel.tsx                       |       0 |        0 |       0 |       0 | 34-52
  index-permission-panel.tsx                         |       0 |        0 |       0 |       0 | 50-176
  role-view.tsx                                      |       0 |        0 |       0 |       0 | 79-379
  tenants-panel.tsx                                  |       0 |        0 |       0 |       0 | 59-186
 public/apps/configuration/panels/tenant-list        |       0 |        0 |       0 |       0 |
  edit-modal.tsx                                     |       0 |        0 |       0 |       0 | 42-101
  tenant-list.tsx                                    |       0 |        0 |       0 |       0 | 57-342
 public/apps/configuration/utils                     |   40.58 |       50 |   30.19 |   40.43 |
  action-groups-utils.tsx                            |       0 |        0 |       0 |       0 | 31-98
  array-state-utils.tsx                              |    4.17 |        0 |       0 |    4.17 | 20-43,73-120
  audit-logging-utils.tsx                            |       0 |      100 |       0 |       0 | 21
  combo-box-utils.tsx                                |      75 |      100 |   66.67 |      75 | 61
  context-menu.tsx                                   |   77.78 |      100 |      50 |    87.5 | 40
  delete-confirm-modal-utils.tsx                     |   54.55 |       25 |      25 |   66.67 | 35-36,41
  display-utils.tsx                                  |    5.56 |        0 |       0 |    5.56 | 36-107,111-131
  form-row.tsx                                       |       0 |        0 |       0 |       0 | 32
  index-permission-utils.tsx                         |      50 |       50 |      50 |      50 | 35
  internal-user-detail-utils.tsx                     |       0 |      100 |       0 |       0 | 21-29
  internal-user-list-utils.tsx                       |       0 |        0 |       0 |       0 | 26-49
  loading-spinner-utils.tsx                          |     100 |      100 |     100 |     100 |
  panel-with-header.tsx                              |       0 |        0 |       0 |       0 | 30
  password-edit-panel.tsx                            |       0 |      100 |       0 |       0 | 25-44
  role-detail-utils.tsx                              |       0 |      100 |       0 |       0 | 21-25
  role-list-utils.tsx                                |      50 |      100 |   57.14 |   42.86 | 90-102
  role-mapping-utils.tsx                             |       0 |        0 |       0 |       0 | 32-59
  storage-utils.tsx                                  |      10 |        0 |       0 |      10 | 24-32,38-44
  tenant-utils.tsx                                   |   84.44 |      100 |   64.71 |   82.93 | 50-54,77-97
  toast-utils.tsx                                    |     100 |      100 |     100 |     100 |
  url-builder.tsx                                    |     100 |      100 |     100 |     100 |
 public/apps/login                                   |       0 |        0 |       0 |       0 |
  login-app.tsx                                      |       0 |      100 |       0 |       0 | 29-30
  login-page.tsx                                     |       0 |        0 |       0 |       0 | 41-125
 public/services                                     |       0 |        0 |       0 |       0 |
  chrome_wrapper.ts                                  |       0 |        0 |       0 |       0 | 19-21
 public/utils                                        |       0 |      100 |       0 |       0 |
  auth-info-utils.tsx                                |       0 |      100 |       0 |       0 | 21
 server                                              |       0 |        0 |       0 |       0 |
  index.ts                                           |       0 |        0 |       0 |       0 | 20-210
  plugin.ts                                          |       0 |        0 |       0 |       0 | 76-160
  types.ts                                           |       0 |        0 |       0 |       0 |
 server/auth                                         |     100 |      100 |     100 |     100 |
  auth_handler_factory.ts                            |     100 |      100 |     100 |     100 |
  user.ts                                            |       0 |        0 |       0 |       0 |
 server/auth/types                                   |       0 |        0 |       0 |       0 |
  authentication_type.ts                             |       0 |        0 |       0 |       0 | 54-209
  index.ts                                           |       0 |        0 |       0 |       0 |
 server/auth/types/basic                             |       0 |        0 |       0 |       0 |
  basic_auth.ts                                      |       0 |        0 |       0 |       0 | 35-134
  routes.ts                                          |       0 |        0 |       0 |       0 | 42-205
 server/auth/types/jwt                               |       0 |        0 |       0 |       0 |
  jwt_auth.ts                                        |       0 |        0 |       0 |       0 | 34-137
  routes.ts                                          |       0 |      100 |       0 |       0 | 27-37
 server/auth/types/openid                            |       0 |        0 |       0 |       0 |
  helper.ts                                          |       0 |        0 |       0 |       0 | 22-65
  openid_auth.ts                                     |       0 |        0 |       0 |       0 | 48-208
  routes.ts                                          |       0 |        0 |       0 |       0 | 32-183
 server/auth/types/proxy                             |       0 |        0 |       0 |       0 |
  proxy_auth.ts                                      |       0 |        0 |       0 |       0 | 35-151
  routes.ts                                          |       0 |        0 |       0 |       0 | 33-77
 server/auth/types/saml                              |       0 |        0 |       0 |       0 |
  routes.ts                                          |       0 |        0 |       0 |       0 | 34-205
  saml_auth.ts                                       |       0 |        0 |       0 |       0 | 35-113
 server/backend                                      |       0 |        0 |       0 |       0 |
  opendistro_security_client.ts                      |       0 |        0 |       0 |       0 | 23-201
  opendistro_security_configuration_plugin.ts        |       0 |        0 |       0 |       0 | 18-216
  opendistro_security_plugin.ts                      |       0 |        0 |       0 |       0 | 18-66
 server/errors                                       |       0 |      100 |       0 |       0 |
  index.ts                                           |       0 |        0 |       0 |       0 |
  no_available_tenant_error.ts                       |       0 |      100 |       0 |       0 | 18
  unauthenticated_error.ts                           |       0 |      100 |       0 |       0 | 18
 server/multitenancy                                 |       0 |        0 |       0 |       0 |
  routes.ts                                          |       0 |        0 |       0 |       0 | 26-129
  tenant_index.ts                                    |       0 |      100 |       0 |       0 | 40-126
  tenant_resolver.ts                                 |       0 |        0 |       0 |       0 | 21-163
 server/routes                                       |       0 |        0 |       0 |       0 |
  index.ts                                           |       0 |        0 |       0 |       0 | 27-828
 server/session                                      |       0 |        0 |       0 |       0 |
  security_cookie.ts                                 |       0 |        0 |       0 |       0 | 45-69
 server/utils                                        |       0 |      100 |       0 |       0 |
  next_url.ts                                        |       0 |      100 |       0 |       0 | 22-25

**After:**
% yarn test:jest --coverage
yarn run v1.22.4
$ NODE_PATH=../../node_modules ../../node_modules/.bin/jest --config ./test/jest.config.js --coverage
 PASS  public/apps/configuration/panels/role-list.test.tsx (19.98s)
 PASS  public/apps/configuration/panels/user-list.test.tsx (19.992s)
 PASS  public/apps/configuration/panels/role-edit/tenant-panel.test.tsx (20.051s)
 PASS  public/apps/configuration/utils/tenant-utils.test.tsx
 PASS  server/auth/auth_handler_factory.test.ts
 PASS  public/apps/configuration/utils/toast-utils.test.tsx
 PASS  public/apps/configuration/utils/role-list-utils.test.tsx
 PASS  public/apps/configuration/panels/role-edit/index-permission-panel.test.tsx
 PASS  public/apps/configuration/panels/role-edit/role-edit.test.tsx
 FAIL  test/jest_integration/basic_auth.test.ts
 PASS  public/apps/configuration/panels/role-edit/cluster-permission-panel.test.tsx

File                                                 | % Stmts | % Branch | % Funcs | % Lines | Uncovered Line #s
-----------------------------------------------------|---------|----------|---------|---------|---------------------------------------------
All files                                            |   13.23 |    11.29 |   14.81 |   12.93 |
 build_tools                                         |       0 |        0 |       0 |       0 |
  build_plugin.js                                    |       0 |        0 |       0 |       0 | 16-153
 common                                              |     100 |      100 |     100 |     100 |
  index.ts                                           |     100 |      100 |     100 |     100 |
 public                                              |       0 |        0 |       0 |       0 |
  index.ts                                           |       0 |      100 |       0 |       0 | 24
  plugin.ts                                          |       0 |        0 |       0 |       0 | 38-122
  types.ts                                           |       0 |        0 |       0 |       0 |
 public/apps                                         |       0 |      100 |     100 |       0 |
  apps-constants.tsx                                 |       0 |      100 |     100 |       0 | 18-21
  types.ts                                           |       0 |        0 |       0 |       0 |
 public/apps/account                                 |       0 |        0 |       0 |       0 |
  account-app.tsx                                    |       0 |        0 |       0 |       0 | 24-40
  account-nav-button.tsx                             |       0 |        0 |       0 |       0 | 45-134
  constants.tsx                                      |       0 |      100 |     100 |       0 | 19
  log-out-button.tsx                                 |       0 |        0 |       0 |       0 | 22-37
  password-reset-panel.tsx                           |       0 |      100 |       0 |       0 | 41-138
  role-info-panel.tsx                                |       0 |        0 |       0 |       0 | 29-73
  tenant-selection-page.tsx                          |       0 |        0 |       0 |       0 | 24-51
  tenant-switch-panel.tsx                            |       0 |        0 |       0 |       0 | 49-249
  types.ts                                           |       0 |        0 |       0 |       0 |
  utils.tsx                                          |       0 |        0 |       0 |       0 | 22-39
 public/apps/configuration                           |   26.87 |        0 |       0 |   27.69 |
  app-router.tsx                                     |       0 |        0 |       0 |       0 | 42-235
  configuration-app.tsx                              |       0 |      100 |       0 |       0 | 30-32
  constants.tsx                                      |     100 |      100 |     100 |     100 |
  cross-page-toast.tsx                               |       0 |        0 |       0 |       0 | 24-34
  types.ts                                           |       0 |        0 |       0 |       0 |
  ui-constants.tsx                                   |     100 |      100 |     100 |     100 |
 public/apps/configuration/panels                    |    61.7 |    54.55 |   31.43 |   64.44 |
  expression-modal.tsx                               |       0 |        0 |       0 |       0 | 28-54
  get-started.tsx                                    |       0 |      100 |       0 |       0 | 37-200
  nav-panel.tsx                                      |       0 |      100 |       0 |       0 | 21-36
  role-list.tsx                                      |   64.86 |       50 |      30 |   66.67 | 55,119-130,143-167,177
  user-list.tsx                                      |   89.47 |    66.67 |   66.67 |   91.89 | 142-177
 public/apps/configuration/panels/audit-logging      |       0 |        0 |       0 |       0 |
  audit-logging-edit-settings.tsx                    |       0 |        0 |       0 |       0 | 45-240
  audit-logging.tsx                                  |       0 |        0 |       0 |       0 | 52-221
  code-editor.tsx                                    |       0 |        0 |       0 |       0 | 27-45
  constants.tsx                                      |       0 |      100 |     100 |       0 | 16-339
  edit-setting-group.tsx                             |       0 |        0 |       0 |       0 | 34-152
  types.tsx                                          |       0 |        0 |       0 |       0 |
  view-setting-group.tsx                             |       0 |        0 |       0 |       0 | 32-62
 public/apps/configuration/panels/auth-view          |       0 |        0 |       0 |       0 |
  auth-view.tsx                                      |       0 |        0 |       0 |       0 | 25-87
  authentication-sequence-panel.tsx                  |       0 |        0 |       0 |       0 | 24-99
  authorization-panel.tsx                            |       0 |        0 |       0 |       0 | 24-87
 public/apps/configuration/panels/internal-user-edit |       0 |        0 |       0 |       0 |
  attribute-panel.tsx                                |       0 |        0 |       0 |       0 | 37-125
  internal-user-edit.tsx                             |       0 |        0 |       0 |       0 | 50-180
  types.tsx                                          |       0 |        0 |       0 |       0 |
 public/apps/configuration/panels/permission-list    |       0 |        0 |       0 |       0 |
  edit-modal.tsx                                     |       0 |        0 |       0 |       0 | 43-97
  permission-list.tsx                                |       0 |        0 |       0 |       0 | 63-321
 public/apps/configuration/panels/permission-tree    |       0 |        0 |       0 |       0 |
  index.ts                                           |       0 |        0 |       0 |       0 |
  permission-tree.tsx                                |       0 |        0 |       0 |       0 | 23-52
 public/apps/configuration/panels/role-edit          |   87.61 |       80 |   89.58 |   87.27 |
  cluster-permission-panel.tsx                       |     100 |      100 |     100 |     100 |
  constant.tsx                                       |     100 |      100 |     100 |     100 |
  index-permission-panel.tsx                         |   93.75 |    83.33 |   90.91 |    93.1 | 271-289
  role-edit.tsx                                      |   79.31 |       75 |   76.92 |   79.31 | 107-111,126-127,140-141,152,155,172-173,249
  tenant-panel.tsx                                   |     100 |      100 |     100 |     100 |
  types.tsx                                          |       0 |        0 |       0 |       0 |
 public/apps/configuration/panels/role-mapping       |       0 |        0 |       0 |       0 |
  RoleEditMappedUser.tsx                             |       0 |        0 |       0 |       0 | 51-162
  external-identities-panel.tsx                      |       0 |        0 |       0 |       0 | 38-106
  internal-users-panel.tsx                           |       0 |      100 |       0 |       0 | 30-31
  types.tsx                                          |       0 |        0 |       0 |       0 |
 public/apps/configuration/panels/role-view          |       0 |        0 |       0 |       0 |
  cluster-permission-panel.tsx                       |       0 |        0 |       0 |       0 | 34-52
  index-permission-panel.tsx                         |       0 |        0 |       0 |       0 | 50-176
  role-view.tsx                                      |       0 |        0 |       0 |       0 | 79-379
  tenants-panel.tsx                                  |       0 |        0 |       0 |       0 | 59-186
 public/apps/configuration/panels/tenant-list        |       0 |        0 |       0 |       0 |
  edit-modal.tsx                                     |       0 |        0 |       0 |       0 | 42-101
  tenant-list.tsx                                    |       0 |        0 |       0 |       0 | 57-342
 public/apps/configuration/utils                     |   41.06 |       50 |   31.13 |   40.43 |
  action-groups-utils.tsx                            |       0 |        0 |       0 |       0 | 31-98
  array-state-utils.tsx                              |    4.17 |        0 |       0 |    4.17 | 20-43,73-120
  audit-logging-utils.tsx                            |       0 |      100 |       0 |       0 | 21
  combo-box-utils.tsx                                |      75 |      100 |   66.67 |      75 | 61
  context-menu.tsx                                   |   88.89 |      100 |      75 |    87.5 | 40
  delete-confirm-modal-utils.tsx                     |   54.55 |       25 |      25 |   66.67 | 35-36,41
  display-utils.tsx                                  |    5.56 |        0 |       0 |    5.56 | 36-107,111-131
  form-row.tsx                                       |       0 |        0 |       0 |       0 | 32
  index-permission-utils.tsx                         |      50 |       50 |      50 |      50 | 35
  internal-user-detail-utils.tsx                     |       0 |      100 |       0 |       0 | 21-29
  internal-user-list-utils.tsx                       |       0 |        0 |       0 |       0 | 26-49
  loading-spinner-utils.tsx                          |     100 |      100 |     100 |     100 |
  panel-with-header.tsx                              |       0 |        0 |       0 |       0 | 30
  password-edit-panel.tsx                            |       0 |      100 |       0 |       0 | 25-44
  role-detail-utils.tsx                              |       0 |      100 |       0 |       0 | 21-25
  role-list-utils.tsx                                |      50 |      100 |   57.14 |   42.86 | 90-102
  role-mapping-utils.tsx                             |       0 |        0 |       0 |       0 | 32-59
  storage-utils.tsx                                  |      10 |        0 |       0 |      10 | 24-32,38-44
  tenant-utils.tsx                                   |   84.44 |      100 |   64.71 |   82.93 | 50-54,77-97
  toast-utils.tsx                                    |     100 |      100 |     100 |     100 |
  url-builder.tsx                                    |     100 |      100 |     100 |     100 |
 public/apps/login                                   |       0 |        0 |       0 |       0 |
  login-app.tsx                                      |       0 |      100 |       0 |       0 | 29-30
  login-page.tsx                                     |       0 |        0 |       0 |       0 | 41-125
 public/services                                     |       0 |        0 |       0 |       0 |
  chrome_wrapper.ts                                  |       0 |        0 |       0 |       0 | 19-21
 public/utils                                        |       0 |      100 |       0 |       0 |
  auth-info-utils.tsx                                |       0 |      100 |       0 |       0 | 21
 server                                              |       0 |        0 |       0 |       0 |
  index.ts                                           |       0 |        0 |       0 |       0 | 20-210
  plugin.ts                                          |       0 |        0 |       0 |       0 | 76-160
  types.ts                                           |       0 |        0 |       0 |       0 |
 server/auth                                         |     100 |      100 |     100 |     100 |
  auth_handler_factory.ts                            |     100 |      100 |     100 |     100 |
  user.ts                                            |       0 |        0 |       0 |       0 |
 server/auth/types                                   |       0 |        0 |       0 |       0 |
  authentication_type.ts                             |       0 |        0 |       0 |       0 | 54-209
  index.ts                                           |       0 |        0 |       0 |       0 |
 server/auth/types/basic                             |       0 |        0 |       0 |       0 |
  basic_auth.ts                                      |       0 |        0 |       0 |       0 | 35-134
  routes.ts                                          |       0 |        0 |       0 |       0 | 42-205
 server/auth/types/jwt                               |       0 |        0 |       0 |       0 |
  jwt_auth.ts                                        |       0 |        0 |       0 |       0 | 34-137
  routes.ts                                          |       0 |      100 |       0 |       0 | 27-37
 server/auth/types/openid                            |       0 |        0 |       0 |       0 |
  helper.ts                                          |       0 |        0 |       0 |       0 | 22-65
  openid_auth.ts                                     |       0 |        0 |       0 |       0 | 48-208
  routes.ts                                          |       0 |        0 |       0 |       0 | 32-183
 server/auth/types/proxy                             |       0 |        0 |       0 |       0 |
  proxy_auth.ts                                      |       0 |        0 |       0 |       0 | 35-151
  routes.ts                                          |       0 |        0 |       0 |       0 | 33-77
 server/auth/types/saml                              |       0 |        0 |       0 |       0 |
  routes.ts                                          |       0 |        0 |       0 |       0 | 34-205
  saml_auth.ts                                       |       0 |        0 |       0 |       0 | 35-113
 server/backend                                      |       0 |        0 |       0 |       0 |
  opendistro_security_client.ts                      |       0 |        0 |       0 |       0 | 23-201
  opendistro_security_configuration_plugin.ts        |       0 |        0 |       0 |       0 | 18-216
  opendistro_security_plugin.ts                      |       0 |        0 |       0 |       0 | 18-66
 server/errors                                       |       0 |      100 |       0 |       0 |
  index.ts                                           |       0 |        0 |       0 |       0 |
  no_available_tenant_error.ts                       |       0 |      100 |       0 |       0 | 18
  unauthenticated_error.ts                           |       0 |      100 |       0 |       0 | 18
 server/multitenancy                                 |       0 |        0 |       0 |       0 |
  routes.ts                                          |       0 |        0 |       0 |       0 | 26-129
  tenant_index.ts                                    |       0 |      100 |       0 |       0 | 40-126
  tenant_resolver.ts                                 |       0 |        0 |       0 |       0 | 21-163
 server/routes                                       |       0 |        0 |       0 |       0 |
  index.ts                                           |       0 |        0 |       0 |       0 | 27-828
 server/session                                      |       0 |        0 |       0 |       0 |
  security_cookie.ts                                 |       0 |        0 |       0 |       0 | 45-69
 server/utils                                        |       0 |      100 |       0 |       0 |
  next_url.ts                                        |       0 |      100 |       0 |       0 | 22-25

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: public/apps/configuration/panels/user-list.test.tsx [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+      jest.spyOn(console, 'log').mockImplementationOnce(loggingFunc);
+      shallow(
+        <UserList
+          coreStart={mockCoreStart as any}
+          navigation={{} as any}
+          params={{} as any}
+          config={{} as any}
+        />
+      );
+      const deleteFunc = useDeleteConfirmState.mock.calls[0][0];
```
Comment (tianleh) [inline public/apps/configuration/panels/user-list.test.tsx:163]: Curious what this syntax does.

File: public/apps/configuration/panels/user-list.test.tsx [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+
+      deleteFunc();
+
+      process.nextTick(() => {
+        expect(loggingFunc).toBeCalled();
+        done();
```
Comment (tianleh) [inline public/apps/configuration/panels/user-list.test.tsx:169]: What is this for?

File: public/apps/configuration/panels/user-list.test.tsx (hunk 1 above)
Comment (tianleh) [reply public/apps/configuration/panels/user-list.test.tsx:163]: Just try to understand the flow. If useDeleteConfirmState is mocked call, how does the handleDelete get called and then requestDeleteUsers?

Comment (yuxitang-amzn) [reply public/apps/configuration/panels/user-list.test.tsx:163]: So `useDeleteConfirmState.mock.calls[0][0]` is the first argument of the first call to the useDeleteConfirmState, in this case it is the handleDelete function

File: public/apps/configuration/panels/user-list.test.tsx (hunk 2 above)
Comment (yuxitang-amzn) [reply public/apps/configuration/panels/user-list.test.tsx:169]: Expect logging the error.

Comment (yuxitang-amzn) [reply public/apps/configuration/panels/user-list.test.tsx:169]: If you mean process.nextTick that is for awaiting the async function.

**Review by tianleh — APPROVED** [review]

**Pull request merged**